src/config/configDefault.js

```diff
@@ -14,6 +14,9 @@ const defaultConfig = {
   // It should be in environment variables (on localhost 'yarn run config' or edit .env file)
   marketplaceRootURL: process.env.REACT_APP_MARKETPLACE_ROOT_URL,
 
+  animal_listing_type: 'animal',
+  acc_listing_type: 'acc',
+
```

Comment:
Those configs should be placed on the configListing.js instead of configDefault.js